### ISSUE TITLE
Inconsistent "const" color in TypeScript inside class

### ISSUE THREAD

**Issue by alexluong** [sdras/night-owl-vscode-theme#237]
Hi,

I recently started working with TypeScript, and I noticed that there is some inconsistency with `const` color when used inside classes.

![Screen Shot 2019-10-16 at 10 12 14 PM](https://user-images.githubusercontent.com/26750127/66974849-22a1c880-f062-11e9-92e9-90f2517b5baa.png)

**Comment by cjke** [comment]
Hit a similar issue - to further illustrate, compare "payload" and "params" in Nightowl (first screenshot) and the standard vscode dark theme (second screenshot):

![image](https://user-images.githubusercontent.com/884065/72256256-d6fcbc00-365c-11ea-8c19-4c7e2ba8f259.png)


![image](https://user-images.githubusercontent.com/884065/72256250-d401cb80-365c-11ea-950e-5047ebdf32b3.png)

Obviously night owl looks better :) - but "payload" should be consistently white

**Comment by MrDrummer** [comment]
It's not that `payload` should be always white- it's that the `const` within classes should be pink as it is outside of a class:

![image](https://user-images.githubusercontent.com/10013572/76352797-8d041f80-6307-11ea-8859-c63aaacb269b.png)

**Comment by cjke** [comment]
@MrDrummer You're right, but I think both are issues (arguably I could've raised a separate issue)

**Comment by jvbianchi** [comment]
Have you tried to set your settings:

`"editor.semanticHighlighting.enabled": false`

**Comment by MrDrummer** [comment]
I should note that it seems to work fine on my linux laptop at work. I freqently keep my work laptop and personal windows PC's settings in sync using an [extension](https://github.com/shanalikhan/code-settings-sync) that uses a private gist to store the settings, extensions and snippets.

Adding the setting you suggested and reloading code didn't change anything.

**Comment by duerdod** [comment]
I have the same issues with classes. I'd love to help debugging this, but I'm not quite sure how.
https://i.imgur.com/IC4InI4.png

**Comment by sfred** [comment]
I published a PR that fixes this issue, please feel free to take a look if you need a fix for this! I have a packaged VSIX extension file available [here](https://github.com/sfred/night-owl-vscode-theme/releases/tag/1.1.4) if you want to install it directly.

![Screen Shot 2020-06-08 at 12 11 54 PM](https://user-images.githubusercontent.com/1349422/84054451-69da2200-a981-11ea-8fd5-7f0467edd310.png)
